### PR TITLE
Update metadata manually for problem collections

### PR DESCRIPTION
**Pull request by zzstoatzz**
`prefect-openmetadata` and `prefect-sqlalchemy` had somewhat strange issues on the cluster that I couldnt quite reproduce locally. Until I can, it should be simple enough to just run these few locally if we really need to (I dont believe these will release often)

### CODE REVIEW

File: src/update_collection_metadata.py [hunk 1]
```diff
@@ -199,6 +199,11 @@ async def update_all_collections(
 
     return Completed(message="All new releases have been recorded.")
 
-if __name__ == "__main__":
-   import asyncio
-   asyncio.run(update_all_collections())
+
+# if __name__ == "__main__":
+## ALL COLLECTIONS
+# asyncio.run(update_all_collections())
```
Comment (desertaxle) [inline src/update_collection_metadata.py:205]: Should this be uncommented?

Comment (zzstoatzz) [reply src/update_collection_metadata.py:205]: I was mostly just trying to make it clear how one could do either locally, I did intentionally comment it out to be explicit that it has no bearing on the automated function of the flow - I can remove and add to the readme if that's better

File: views/aggregate-block-metadata.json [hunk 2]
```diff
@@ -8309,7 +8309,7 @@
       "database-credentials": {
         "name": "Database Credentials",
         "slug": "database-credentials",
-        "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/fb3f4debabcda1c5a3aeea4f5b3f94c28845e23e-250x250.png",
+        "logo_url": "https://images.ctfassets.net/gm98wzqotmnx/3xLant5G70S4vJpmdWCYmr/8fdb19f15b97c3a07c3af3efde4d28fb/download.svg.png?h=250",
```
Comment (desertaxle) [inline views/aggregate-block-metadata.json:8312]:
```suggestion
        "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/fb3f4debabcda1c5a3aeea4f5b3f94c28845e23e-250x250.png",
```

File: views/aggregate-block-metadata.json [hunk 3]
```diff
@@ -8426,13 +8426,13 @@
...
           "capabilities": [],
-          "version": "0.2.4"
+          "version": "0.3.0"
         }
       },
       "sqlalchemy-connector": {
         "name": "SQLAlchemy Connector",
         "slug": "sqlalchemy-connector",
-        "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/3c7dff04f70aaf4528e184a3b028f9e40b98d68c-250x250.png",
+        "logo_url": "https://images.ctfassets.net/gm98wzqotmnx/37TOcxeP9kfXffpKVRAHiJ/0f359112e79d0bd3dfe38c73c4fc6363/sqlalchemy.png?h=250",
```
Comment (desertaxle) [inline views/aggregate-block-metadata.json:8435]:
```suggestion
        "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/3c7dff04f70aaf4528e184a3b028f9e40b98d68c-250x250.png",
```

File: src/update_collection_metadata.py (hunk 1 above)
Comment (desertaxle) [reply src/update_collection_metadata.py:205]: Oh gotcha, makes sense!

**Review by desertaxle — APPROVED** [review]

**Pull request merged**